save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -136,12 +141,26 @@ class DockerService(private val configProperties: ConfigProperties) {
     }
 
     @Suppress("TOO_LONG_FUNCTION", "UnsafeCallOnNullableType")
-    private fun buildBaseImageForExecution(execution: Execution): Pair<String, String> {
+    private fun buildBaseImageForExecution(execution: Execution, testSuiteDtos: List<TestSuiteDto>?): Triple<String, String, String> {
         val resourcesPath = File(
             configProperties.testResources.basePath,
             execution.resourcesRootPath,
         )
-        val runCmd = "./$SAVE_AGENT_EXECUTABLE_NAME"
+        val agentRunCmd = "./$SAVE_AGENT_EXECUTABLE_NAME"
+
+        // collect standard test suites for docker image, which were selected by user, if any
+        val testSuitesForDocker = collectStandardTestSuitesForDocker(testSuiteDtos)
+        val testSuitesDir = resourcesPath.resolve(standardTestSuiteDir)
+        // copy corresponding standard test suites to resourcesRootPath dir
+        copyTestSuitesToResourcesPath(testSuitesForDocker, testSuitesDir)
+        val saveCliExecFlags = if (testSuitesForDocker.isNotEmpty()) {
+            // create stub toml config in aim to execute all test suites directories from `testSuitesDir`
+            testSuitesDir.resolve("save.toml").apply { createNewFile() }.writeText("[general]")
```

Comment:
Why do we need it here? Can it already exist in the standard suites repo?

Reply:
The logic was the following:
Each standard test suite have own `save.toml` and own configuration. So they should be cloned into different places.
I created the base directory for standard test suites:  `standard-test-suites`

In aim to distinguish each test suite, I also created the separate directory for each of suite: `DocksCheck_131231_213131`, `autofix_21312_12313` in `standard-test-suites`.

However I would like to execute all suites at the same time. So I create some stub `save.toml` in `standard-test-suites`
and would like to execute SAVE like `./save standard-test-suites`

Reply:
Ok, I see